NMap Scanner/README.md

```diff
@@ -0,0 +1,38 @@
+# Nmap Scanner
+Nmap(Network Mapper) is a security scanner, used to discover hosts and services on a computer network, thereby building a map of the network. To accomplish its goal, Nmap sends specially crafted packets to the target host(s) and then analyzes their responses.
+
+## Prerequisites
+
+This requires installing nmap in our local machine.
+1. If you're using Linux, then use:
+```
+sudo apt-get install nmap
+```
+This is for Ubuntu.  
+
+2. If you're using Windows, then download nmap from this ![link](https://nmap.org/download.html).
```

Comment:
This Markdown Syntax is used to refer to Images, not Links. Kindly rectify this.